templates/settings-admin.php

```diff
@@ -304,15 +294,15 @@ class="barchart"
 				<div class="infobox">
 					<div class="database-wrapper">
 						<p>
-							<?php p($l->t('Type:')); ?>
+							<?php p($l->t('Type').':'); ?>
```

Comment:
The colon should always be part of the translated string, as it may be changed by the translation.
For instance in French a (small nonbreakable) space is expected before a double punctuation like a colon.

Reply:
Yes, you are absolutely right 👍, but in this case an exception is made, because using a colon in this PHP code with an entire text string means that the entire text string in the application is not displayed 😬. This was one of the problems that this change arose. I haven't found any other sensible solution to this problem, which doesn't mean it needs to be corrected.
I would like to ask developers who know more about PHP programming to look at this code, and maybe they will find a solution, and will be happy to implement it, so that French translators can correctly translate these text strings.

@nickvergessen could you have a look at it and advise us something?

Reply:
I already mentioned it here that this is necessary: https://github.com/nextcloud/serverinfo/pull/333/files#r752009811

Reply:
@nickvergessen Yes, I know this and I remember it, but after using a colon text string, I don't know why, but this text string is not displayed at all. This was a problem before. Can't you use a colon in a text string in PHP?

Reply:
Fix https://github.com/nextcloud/server/pull/32155